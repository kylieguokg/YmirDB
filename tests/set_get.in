set
set aa 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28
get aa
set AA 19 20 392 
get AA
set aA 0
get aA
set Aa 1
get Aa
list entries
BYE
