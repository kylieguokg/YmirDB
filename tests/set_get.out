> invalid key

> ok

> [1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28]

> ok

> [19 20 392]

> ok

> [0]

> ok

> [1]

> Aa [1]
aA [0]
AA [19 20 392]
aa [1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28]

> bye
